src/comps/main/addComments/addComments.jsx

```diff
@@ -14,43 +16,77 @@ export const AddComments = props => {
     });
   }
 
+  // handle submitted form
   function handleSubmit(e) {
     e.preventDefault();
-    // console.log(comment.hasOwnProperty("comment"));
+
     if (false) {
       console.log("cant send, the comment is not valid");
     } else {
       addCommentFunc(comment)
-        .then(res => alert("added new comment..."))
+        .then(res => {
+          alert("added new comment...");
+          getComments();
+        })
         .catch(err => console.log(err));
     }
   }
 
+  function getComments() {
```

Comment:
This doesn't just "get" the comments, it also sets them to state,
consider naming this "loadComments" or "updateComments"